dataset-variance-resub-2022/dataset.json

```diff
@@ -0,0 +1,17 @@
+{
+    "title": "hiPSC Single-Cell Image Dataset",
+    "name": "In revision",
```

Comment:
This isn't a good name for the dataset. The name is supposed a unique key we use to store the data

Reply:
The official name of this dataset is **WTC-11 hiPSC Single-Cell Image Dataset v1**.